man/fabtests.7.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: page
+title: fabtests(7)
+tagline: Fabtests Programmer's Manual
+---
+
+## NAME
+
+Fabtests
+
+##SYNOPSIS
+
+Fabtests is a set of examples for fabric providers that demonstrates various features of libfabric- high-performance fabric software library.
+
+## OVERVIEW
+  
+Libfabric defines sets of interface that fabric providers can support. The purpose of Fabtests examples is to demonstrate some of the major features. The goal is to familiarize users with different functionalities libfabric offers and how to use them.
+
+The tests are divided into four categories:
+
+### Simple
+
+These tests are a mix of very basic tests and major functionalies of libfabric. All of the tests except info.c are designed to run as client-server processes. A server is satrted first and then a client process connects to the server and performs various operations.
```

Comment:
"functionalies" -> "functionalities" (consider changing this to "features")
"satrted" -> "started"